lib/house.rb

```diff
@@ -0,0 +1,54 @@
+class House
+
+  attr_reader :address, :price, :rooms
+
+  def initialize(price, address)
+    @price = price[1..-1].to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    rooms << room
+  end
+
+  def above_market_average?
+    price > 500000
+  end
+
+  def rooms_from_category(category)
+    rooms.find_all { |room| room.category == category }
+  end
+
+  def area
+    total_area = []
+    rooms.each { |room| total_area << room.area }
+    total_area.sum
+  end
+
+  def details
+    Hash.new
+    {"price" => price, "address" => address}
+  end
+
+  def price_per_square_foot
+    (price.to_f / area.to_f).round(2)
```

Comment:
nice job using the return value of another method here.  Tiny point, but you only need one of those values to be a float in order to get a float in your answer, so you only need to do `to_f` on either price or area, but not both.